hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java

```diff
@@ -402,6 +408,8 @@ private ContainerCommandResponseProto dispatchRequest(
         audit(action, eventType, params, AuditEventStatus.FAILURE,
             new Exception(responseProto.getMessage()));
       }
+      perf.appendOpLatencyMs(oPLatencyMS);
+      performanceAudit(action, params, perf);
```

Comment:
Thanks for the patch @xichen01. One comment here, I think the performance should be audited if the opLatency `isExceedThreshold()`. Let me know if that makes sense. Otherwise, LGTM

Reply:
Yes, have added.